design-system/materials/internals/definition.yaml

```diff
@@ -414,11 +477,23 @@ decisionGroupsByTheme:
         font-weight-for-text-as-detail:
           description: ''
           choice: font-weight-400
+        font-weight-for-button:
+          choice: font-weight-400
 
     shadows:
       label: Shadows
       prefix: shadow
       decisions:
+        shadow-for-button:
+          choice: shadow-7
+        shadow-for-button-when-focused:
+          choice: shadow-8
+        shadow-for-button-when-hovered:
+          choice: shadow-8
+        shadow-for-button-when-active:
+          choice: shadow-9
+        shadow-for-button-when-disabled:
+          choice: '0 0 0 1px var(--color-neutral) inset'
```

Comment:
Currently we are using this value, but for the new theme we don't want any shadow so I need to use the whole value as a choice to be able to have then `none` value in the new theme choice.